plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetReaderConfig.java

```diff
@@ -59,6 +59,19 @@ public ParquetReaderConfig setMaxReadBlockSize(DataSize maxReadBlockSize)
         return this;
     }
 
+    @NotNull
```

Comment:
`@NotNull` is not needed since it's a primitive type now.
We should also have `@Min(128)` and `@Max(65536)` to keep things in reasonable range.